docs/docker-deployment.md

```diff
@@ -1,13 +1,21 @@
-# Docker deployement
+# Docker deployment
 
 This documents the [Docker][] way to deploy uData.
 If you are intending to perform a classic deployment on a server, you should look at the [Installation section](installation.md).
 
-uData and its extensions are packaged as [Docker image][]
-You can go directly to [the dedicated repository][docker-repository] for an up to date docuementation
-and some sample configuration using [Docker Compose][].
+uData and its extensions are packaged and distributed as [Docker images][Docker image].
```

Comment:
Yeah, my bad, I hesitated to change the link reference for too long and mixed it up.
Will update accordingly.